src/systems/physics/Physics.cc

```diff
@@ -1458,14 +1458,6 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
         if (modelIt == this->entityModelMap.end())
           return true;
 
-        // world pose cmd currently not supported for nested models
-        if (_entity != topLevelModel(_entity, _ecm))
```

Comment:
I'm not sure TPE supports moving nested models separately from their top level model. So, maybe we shouldn't remove this yet?

Reply:
Restored

Reply:
Hmm, if you have this check then a nested model won't transform correctly, even if it's fully connected by joints.